kroxylicious/src/main/java/io/kroxylicious/proxy/internal/codec/KafkaRequestDecoder.java

```diff
@@ -85,12 +97,19 @@ protected Frame decodeHeaderAndBody(ChannelHandlerContext ctx, ByteBuf in, final
 
         RequestHeaderData header = null;
         final ByteBufAccessorImpl accessor;
+        Metrics.counter(kroxylicious_inbound_downstream_messages, List.of(Tag.of("flowing", "downstream"))).increment();
         var decodeRequest = decodePredicate.shouldDecodeRequest(apiKey, apiVersion);
         LOGGER.debug("Decode {}/v{} request? {}, Predicate {} ", apiKey, apiVersion, decodeRequest, decodePredicate);
         boolean decodeResponse = decodePredicate.shouldDecodeResponse(apiKey, apiVersion);
         LOGGER.debug("Decode {}/v{} response? {}, Predicate {}", apiKey, apiVersion, decodeResponse, decodePredicate);
         short headerVersion = apiKey.requestHeaderVersion(apiVersion);
         if (decodeRequest) {
+            Metrics.counter(kroxylicious_inbound_downstream_decoded_messages, List.of(Tag.of("flowing", "downstream"))).increment();
+            Metrics.summary(kroxylicious_request_size, List.of(
```

Comment:
keen to have the unit in here like `kroxylicious_request_size_bytes`

Reply:
done!